coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/metric/MetricRegistryBean.java

```diff
@@ -0,0 +1,7 @@
+package org.wickedsource.coderadar.metric;
+
+import com.codahale.metrics.MetricRegistry;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MetricRegistryBean extends MetricRegistry {}
```

Comment:
what do we need this bean for?

Reply:
Coderadar requires a MetricRegistry bean to be available, but it is no longer automatically created, therefore I created it manually.

Reply:
OK